notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>Can we just leave a <code>pip install</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Yes, removing this block.</p>